test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -535,6 +521,28 @@ def __init__(self, out_dir, diag_path=None, server_host=None,
         # Create the snapshot directory tree
         self.__initialize_snapshot()
 
+        # Create a PBSAnonymizer object
+        self.anonymize = anonymize
+        self.custom_rscs = self.server.parse_resources()
+        if self.anonymize:
+            del_attrs = [ATTR_v, ATTR_e, ATTR_mailfrom, ATTR_m, ATTR_name,
+                         ATTR_jobdir, ATTR_submit_arguments, ATTR_o, ATTR_S]
+            obf_attrs = [ATTR_euser, ATTR_egroup, ATTR_project, ATTR_A,
+                         ATTR_operators, ATTR_managers, ATTR_g, ATTR_M,
+                         ATTR_u, ATTR_SvrHost, ATTR_aclgroup, ATTR_acluser,
+                         ATTR_aclResvgroup, ATTR_aclResvuser, ATTR_SchedHost,
+                         ATTR_aclResvhost, ATTR_aclhost, ATTR_owner,
+                         ATTR_exechost, ATTR_NODE_Host, ATTR_NODE_Mom,
+                         ATTR_rescavail + ".host", ATTR_rescavail + ".vnode"]
+            obf_rsc_attrs = []
+            if self.custom_rscs is not None:
+                for rsc in self.custom_rscs.keys():
+                    obf_rsc_attrs.append(rsc)
```

Comment:
Hmm .. I'm not sure if that's a requirement (anonymize all custom resource attributes with their values), how about we wait until a user requests it?